examples/key_detector.py

```diff
@@ -0,0 +1,64 @@
+"""
+Arcade Key Detector
+
+Shows arcade key constants for pressed keys.
+"""
+
+import arcade
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 500
+TITLE = "Arcade Key Detector"
+BACKGROUND_COLOR = arcade.color.WHITE
+TEXT_COLOR = arcade.color.BLACK
+FRAMES_PER_SECOND = 25
+LINE_HEIGHT = 40
+
+
+def load_arcade_key_dict():
+    file = open(arcade.key.__file__)
+    result = dict()
+    for line in file:
+        parts = line.replace('\n', '').replace(' ', '').split('=')
+        if len(parts) == 2:
+            key = int(parts[1])
+            name = parts[0]
+            if key in result:
+                result[key] = result[key] + ', ' + name
+            else:
+                result[key] = name
+
+    file.close()
+    return result
+
+
+class MyWindow(arcade.Window):
+    def __init__(self):
+        super().__init__(SCREEN_WIDTH, SCREEN_HEIGHT, TITLE)
+        self.key_dict = load_arcade_key_dict()
+        arcade.set_background_color(BACKGROUND_COLOR)
+
+
+    def on_draw(self):
+        arcade.start_render()
+        y = SCREEN_HEIGHT - LINE_HEIGHT
+        for pressed_key in self.pressed_keys:
```

Comment:
Where does self.pressed_keys come from? It errors out for me on the line above, not knowing what it is. Is this in a different version of pyglet than I have?